gulpfile.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```

Comment:
I believe our policy is to not update the copyright year of existing files

Reply:
I made an exception here because `gulpfile.js` is from 2015, not 2017, and the file has been virtually rewritten this week.